test/bigtest/network/config.js

```diff
@@ -9,4 +11,9 @@ export default function config() {
     jobExecutions: [],
     totalRecords: 3,
   }, 200);
+
+  this.get('/data-export/jobExecutions/:jobLogId/download/:fileId', {
+    fileId: '448ae575-daec-49c1-8041-d64c8ed8e5b1',
+    mockLink,
```

Comment:
As far as I understand, the current solution handles 'link' field. Do we use a field mockLink anywhere?

Reply:
Good catch